persistence/gorm/gorm.go

```diff
@@ -61,13 +62,15 @@ type storage struct {
 // DefaultTypes contains a list of internal types that need to be migrated by default
 var DefaultTypes = []any{
 	&auth.User{},
-	&assessment.MetricConfiguration{},
 	&orchestrator.CloudService{},
 	&assessment.MetricImplementation{},
 	&assessment.Metric{},
 	&orchestrator.Certificate{},
 	&orchestrator.State{},
-	&orchestrator.Requirement{},
+	&orchestrator.Catalog{},
+	&orchestrator.Category{},
+	&orchestrator.Control{},
+	&orchestrator.TargetOfEvaluation{},
```

Comment:
since TargetOfEvaluation is a join table, it is not necessary to auto-migrate it. Not sure if there is any harm in doing it.